lib/transform/config.js

```diff
@@ -29,6 +29,7 @@ var MISSING_TRANSFORM_ERRORS = GroupMessages.GROUP.MissingRegisterTransformError
  */
 function transformConfig(config, dictionary, platformName) {
   var to_ret = _.clone(config);
+  to_ret.platformName = platformName;
```

Comment:
Is this being used elsewhere? I can't find this being used in this PR.

Reply:
No it is not. I was using it to generate the message originally, but then realized I could just use the current output stream instead of generating something unique.

I thought it made sense to have the platform name present in the platform (that data is currently not available), so I left it. I can pull to a different PR if you want.

Reply:
Yea let's pull it to a different PR, just so this one is focused on a single problem.

Reply:
removed